grails-forge-core/src/main/resources/pom.xml

```diff
@@ -125,12 +125,12 @@
         <dependency>
             <groupId>com.bertramlabs.plugins</groupId>
             <artifactId>asset-pipeline-grails</artifactId>
-            <version>4.3.0</version>
+            <version>4.5.1</version>
         </dependency>
         <dependency>
             <groupId>com.bertramlabs.plugins</groupId>
             <artifactId>asset-pipeline-gradle</artifactId>
-            <version>4.3.0</version>
+            <version>4.5.1</version>
         </dependency>
```

Comment:
`4.5.2` will default to `skipNonDigests = true` which means that it can be removed from the `assets` block in `build.gradle`.
`minifyJs` and `minifyCss` are already defaulting to `true`, so that means the entire `assets` block can be removed (if we don't want to explicitly show it in the build file).

Reply:
4.5.2 has not been released to https://plugins.gradle.org/plugin/com.bertramlabs.asset-pipeline, so going back to 4.5.1 for this PR

`Plugin [id: 'com.bertramlabs.asset-pipeline', version: '4.5.2'] was not found in any of the following sources:`

Reply:
https://github.com/bertramdev/asset-pipeline/issues/358

Reply:
Hmm, it is available here https://mvnrepository.com/artifact/com.bertramlabs.plugins/asset-pipeline-gradle

I wonder if something went wrong or if it takes longer to show up in the Gradle Plugin Portal.
@davydotcom Do you know, why it's not available yet?